wps_tools/utils.py

```diff
@@ -170,3 +170,21 @@ def log_handler(
     log_file_path = Path(process.workdir) / log_file_name  # From Finch bird
     log_file_path.open("a", encoding="utf8").write(message + "\n")
     response.update_status(message, status_percentage=status_percentage)
+
+
+def copy_http_content(http, tmp_file):
+    """
+        This function is implemented to copy the content of a file passed
+        as an http address to a tmporary file.
+
+        Parameters:
+            http (str): http address containing the desired content
+            tmp_file (tempfile._TemporaryFileWrapper): path to the 
+                temporary file that the content will be copied to
+        Returns:
+            Path to the copied file in /tmp directory
+        """
+    http_content = get(http).content
+    tmp_file.write(http_content)
+
+    return tmp_file.name
```

Comment:
It isn't much, but this saves us from repeating these lines over and over in different birds and demos. If you wanted to be fancy you could squish some of these lines together. It looks good as is though.